### ISSUE TITLE
runtime: cannot run GO in Windows Server 2016

### ISSUE THREAD

**Issue by Westwoodmayim** [golang/go#21060]
[Issue]
When running go, a problem is encountered

```
C:\Users\Administrator>go
Exception 0xc0000005 0x8 0x0 0x0
PC=0x0

runtime.asmstdcall(0x8fdd8, 0x40ee8d, 0x10f4000, 0x0, 0x100000044f980, 0x20, 0x11, 0x10f4000, 0x44f8c0, 0x10f0000, ...)
        c:/go/src/runtime/sys_windows_amd64.s:60 +0x5e fp=0x8fd80 sp=0x8fd70
rax     0x0
rbx     0xa992c0
rcx     0xab4e20
rdi     0x204000
rsi     0x8fea0
rbp     0x8fe68
rsp     0x8fd68
r8      0x42e36e
r9      0x8fee0
r10     0xc042000000
r11     0xffffffff
r12     0x0
r13     0xffffffee
r14     0x0
r15     0x0
rip     0x0
rflags  0x10293
cs      0x33
fs      0x53
gs      0x2b
```

[Environment]

Version:
Go 1.8.3 From go1.8.3.windows-amd64.msi (newest).
And the issue resists in go1.9beta2 (newest unstable)

OS:
Microsoft Windows Server 2016 Datacenter

Hardware:
ASUSTek COMPUTER INC. Z10PA-D8 Series
Intel(R) Xeon(R) CPU E5-2620 v4 @2.10GHz, Intel(R) Xeon(R) CPU E5-2620 v4 @2.10GHz
128GB RAM
512GB SSD + 2TB HHD + 6TB HHD

**Comment by bradfitz** [comment]
We test on Windows 2008, 2012, and 2016. It works on all ours.

What's different with your machine?

Are you running any anti-virus, by chance?

**Comment by zergos** [comment]
Hello. I approve that. Go version 1.9.1.
Problem appeared after first run on clean install.
Windows 2016 Standard x64
No Antiviral or any other active protection installed.

```
Exception 0xc0000005 0x8 0x0 0x0
PC=0x0

runtime.asmstdcall(0x41051d, 0x1194000, 0x0, 0x454f25, 0x411580, 0x23, 0x1194000, 0x451e90, 0x1190000, 0x4023, ...)
        c:/go/src/runtime/sys_windows_amd64.s:60 +0x5e fp=0xfffd80 sp=0xfffd70 pc=0x4589de
rax     0x0
rbx     0xad9f18
rcx     0xaf6c20
rdi     0x25e000
rsi     0xfffea0
rbp     0xfffe68
rsp     0xfffd68
r8      0x42f15d
r9      0xfffee0
r10     0x0
r11     0x246
r12     0x7c588b
r13     0x0
r14     0x0
r15     0x0
rip     0x0
rflags  0x10293
cs      0x33
fs      0x53
gs      0x2b

````

Have no idea, what possible difference do we have there. But it has started successfully for the first time. And it was last time...
Guys, help.

**Comment by zergos** [comment]
After little investigation I found 60th line of `c:/go/src/runtime/sys_windows_amd64.s`:
```
	// Call stdcall function.
	CALL	AX
```
and it run with rax=0x0
hm... what does it call? Сthulhu?

**Comment by alexbrainman** [comment]
> CALL	AX
> and it run with rax=0x0

Yes, AX should not be 0. You should check whole stack trace, not just the last fragment to discover how AX got to 0.

BTW this issues is closed, and we do not comment on closed issues. If you have some bug to report, please, create new issue. But you would need to provide much more information.

Alex

**Comment by alexbrainman** [comment]
Another similar issue is #40469. Perhaps we need PC with 128 GB of memory to reproduce this.

Reopening this issue, so we don't forget to check.

Alex
